_posts/2019-07-29-hhvm-4.16.0.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: "HHVM 4.16.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.16 is released! This release marks the end of support for 4.10;
+4.11-4.15 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- typechecker error messages now suggest similar methods and variables
+- reification of `dynamic`, functions and opaque type aliases is now allowed (formerly typing error 4305)
```

Comment:
probably best to remove this, and either document reifiy as a whole next week, or do a standalone post in the next few days introducing it.

`reify` as a whole is not announced yet